proto/beacon/p2p/v1/types.proto

```diff
@@ -145,12 +145,28 @@ message ShardAndCommittee {
 }
 
 message BeaconBlock {
-  repeated bytes ancestor_hashes = 1;
-  uint64 slot = 2;
-  bytes randao_reveal = 3;
-  bytes pow_chain_ref = 4;
-  bytes state_root = 5;
-  google.protobuf.Timestamp timestamp = 6;
-  repeated AggregatedAttestation attestations = 7;
-  repeated SpecialRecord specials = 8;
+  uint64 slot = 1;
```

Comment:
Can we use ranges here as well?

Reply:
Mentioned offline, but we only need ranges for large data protos. For messages that we believe will be small (less than 10 fields) then assigning ID block ranges is not necessary.